linode/manager
# change: SD-598 - Ticket page small refresh

## Description 📝
The changes were requested from our support team, these changes would allow us to have an updated ticket page with the regards with a minor tweak in the wording of for the Akamai employee roles. 

## Changes  🔄
- _Changed Linode Expert_ -> Customer Support
- Update Emotes for How did I do on the ticket page.

## Preview 📷

| Before  | After   |
| ------- | ------- |
| ![image](https://github.com/linode/manager/assets/130775539/7d70b3a0-b0e2-4ab6-b5cd-5ad43fc7b261) | ![image](https://github.com/linode/manager/assets/130775539/8f78bdaa-5a18-445f-a97d-225833d339bd) |
| ![image](https://github.com/linode/manager/assets/130775539/f8460078-4d95-4d53-8dc1-38bc02360a0c) | ![image](https://github.com/linode/manager/assets/130775539/929ed3d2-0aea-4729-8f7c-d1b08ae8b75a) |

## How to test 🧪

### Verification steps 
- To test open the ticket page as a normal user, and open a support ticket, once you receive a reply you will be able to see the updated names as well as the updated "How did I do?" emotes.
-  To view changes please reference the before and after photos provided above for each change.


## As an Author I have considered 🤔
